Designer/ViewModel/ViewDesignViewModel.cs

```diff
@@ -463,6 +507,11 @@ public class ViewDesignViewModel : INotifyPropertyChanged {
             selectScreen.DeleteButton.Click += delegate {
                 ProductPlacements.Remove(placement);
                 _selectedPlacement = null;
+                RenderRoom();
```

Comment:
Deze moeten hier niet meer aangeroepen worden, weet niet zeker of je dit express doet